Page Object/Elements/buttonsPage.ts

```diff
@@ -3,4 +3,17 @@ export class ButtonsPage extends basePage {
     constructor (page) {
         super (page)
     }
+
+    Buttons = {
+        buttonsButton: this.page.getByText("Buttons"),
+        doubleClickButton: this.page.locator('#doubleClickBtn'),
+        rightClickButton: this.page.locator('#rightClickBtn'),
+        clickButton: this.page.getByRole('button', { name: 'Click Me', exact: true }),
```

Comment:
Is it necessary to use `exact: true` in this situation?

Reply:
Yes, bcs we have at least 2 more buttons with "Click me".